torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4411,7 +4411,11 @@ def _maybe_evaluate_static(
                 # Skip var_ranges logic for SingletonInt which is only used
                 # for jagged layout NestedTensors today
                 continue
-            vr = var_ranges[k]
+            try:
+                vr = self.var_to_range[k]
+            except KeyError:
+                log.warning("{} is not in var_to_range, defaulting to unknown range.".format(k))
```

Comment:
```suggestion
                log.warning("%s is not in var_to_range, defaulting to unknown range.", k)
```

Reply:
thx! was looking for the proper way to do that